### PR TITLE
move incubator/osg-hosted-ce to stable (SOFTWARE-4564)

### PR DESCRIPTION
**Pull request by edquist**
in collaboration with @brianhlin

### CODE REVIEW

**Review by brianhlin — DISMISSED** [review]
@edquist and I worked on this together so this LGTM

**Comment by Mansalu** [comment]
Needs another chart version bump. `3.8.1` collides with the version from the last PR.

File: stable/osg-hosted-ce/osg-hosted-ce/Chart.yaml [hunk 1]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "4.4.1"
 description: OSG Hosted Compute Element
 name: osg-hosted-ce
-version: 3.7.4
+version: 3.8.1
```
Comment (Mansalu) [inline stable/osg-hosted-ce/osg-hosted-ce/Chart.yaml:5]:
```suggestion
version: 3.8.2
```

**Review by Mansalu — CHANGES_REQUESTED** [review]

**Comment by brianhlin** [comment]
But 3.8.1 is associated with the incubator version. Do the stable/incubator versions have to be distinct?

@edquist one other thing I realized that we're missing is a commented out section in the stable values file for the SciTokens mapping. For a commented out default, please add:

```
# some helpful text here
# SciTokenRemoteUserMapping:
#   - "https://scitokens.org/osg-connect": osg01
```

**Comment by Mansalu** [comment]
This PR appears to touch the incubator version as well as the stable one.

**Comment by edquist** [comment]
>This PR appears to touch the incubator version as well as the stable one.

**Comment by Mansalu** [comment]
You'll either need to remove the incubator changes *or* update the chart version within the incubator `Chart.yaml`. Whichever makes the most sense.

File: stable/osg-hosted-ce/osg-hosted-ce/values.yaml [hunk 2]
```diff
@@ -103,6 +103,10 @@ VoRemoteUserMapping:
 #DnRemoteUserMapping:
 #  - "/DC=foo/DC=bar/OU=Organic Units/OU=Users/CN=YourUserName": osg01
 
+# SCITOKENS url to user mapping
+# SciTokenRemoteUserMapping:
+#   - "https://scitokens.org/osg-connect": osg01
```
Comment (brianhlin) [inline stable/osg-hosted-ce/osg-hosted-ce/values.yaml:108]:
```suggestion
    # https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/OSG.yaml
    # - "https://scitokens.org/osg-connect": osg01
    # https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/GLOW.yaml
    # - "https://chtc.cs.wisc.edu": osg02
    # https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/CMS.yaml
    # - "https://cms-auth.web.cern.ch/": osg04
    # https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/ATLAS.yaml
    # - "https://atlas-auth.web.cern.ch/": osg09
```

Fleshing out the commented issuers

**Comment by brianhlin** [comment]
> This PR appears to touch the incubator version as well as the stable one.

Ah right. @edquist go ahead and bump the incubator chart version, too

**Comment by edquist** [comment]
Updated, lmk if anything else is needed

**Review by Mansalu — APPROVED** [review]
LGTM. Thanks for your patience! 😄

**Pull request merged**